app/templates/gentelella/users/settings/pages/applications.html

```diff
@@ -14,13 +14,12 @@
 {% block content %}
 
     <div style="display: block;">
-        {% if user.user_detail.facebook %}
         <div class="row" style="margin-bottom:20px">
 
             <div class="col-md-6 col-xs-6">
                 <label>Facebook:</label>
                 <div class="input-group">
-                    <span class="input-group-addon">facebook.com/</span>
+                    <span class="input-group-addon" id="basic-addon3">facebook.com/</span>
```

Comment:
The class is also being used in the search button and I checked the ids are only used in the span tags. Should I just change the property of each span tag?